### PR TITLE
Added flag for drag and drop in Open with tutorial

### PR DESCRIPTION
**Pull request by kikettas**
### Description
Now the drag and drop step within the open with tutorial is optional.

### How to test
Change the `shouldShowDragAndDropTutorial` in the `GiniConfiguration` to false and check that the last step is not shown.

### Merging
Automatic

### CODE REVIEW

**Comment by kikettas** [comment]
[Add toggle for Drag&Drop tip in OpenWith tutorial and release update 3.3.5](https://trello.com/c/miD1mW4w/5-add-toggle-for-dragdrop-tip-in-openwith-tutorial-and-release-update-335)

File: Example/Tests/GINIOpenWithTutorialViewControllerTests.swift [hunk 1]
```diff
@@ -121,4 +121,17 @@ class GINIOpenWithTutorialViewControllerTests: XCTestCase {
         XCTAssertEqual(headerSize.height, 0, "header size should be 0 on landscape mode")
         
     }
+    
+    func testItemsWhenDragAndDropTipNotAppear() {
```
Comment (nsobadjiev) [inline Example/Tests/GINIOpenWithTutorialViewControllerTests.swift]: DoesNotAppear*

File: Example/Tests/GINIOpenWithTutorialViewControllerTests.swift [hunk 2]
```diff
@@ -121,4 +121,17 @@ class GINIOpenWithTutorialViewControllerTests: XCTestCase {
...
+        let giniConfiguration = GiniConfiguration()
+        giniConfiguration.shouldShowDragAndDropTutorial = false
+        let openWithTutorialViewController = OpenWithTutorialViewController(giniConfiguration: giniConfiguration)
+        
+        _ = openWithTutorialViewController.view
+        let collectionSection0ItemsCount = openWithTutorialViewController
+            .collectionView(openWithTutorialViewController.collectionView!, numberOfItemsInSection: 0)
+        
+        XCTAssertEqual(2, collectionSection0ItemsCount,
```
Comment (nsobadjiev) [inline Example/Tests/GINIOpenWithTutorialViewControllerTests.swift]: The fact that there are two items doesn't mean they are the right ones. You can, for instance, check if the Drag & Drop one is NOT there.

File: GiniVision/Classes/GiniConfiguration.swift [hunk 3]
```diff
@@ -667,4 +667,9 @@ import UIKit
      Sets the color of the warning container background to the specified color
      */
     public var noResultsWarningContainerIconColor = Colors.Gini.rose
+    
+    /**
+     Sets if the Drag&Drop step should be shown in the open with tutorial list
```
Comment (nsobadjiev) [inline GiniVision/Classes/GiniConfiguration.swift]: I'd write open with as "Open with" (with the quotes). So:

> Sets if the Drag&Drop step should be shown in the "Open With" tutorial <strike>list</strike>

File: Example/Tests/GINIOpenWithTutorialViewControllerTests.swift (hunk 2 above)
Comment (kikettas) [reply Example/Tests/GINIOpenWithTutorialViewControllerTests.swift]: That makes more sense 👍

**Pull request merged**